.clang-format

```diff
@@ -18,15 +21,16 @@ BraceWrapping:
   AfterExternBlock: true
   BeforeCatch: true
   BeforeElse: true
-  IndentBraces: true
   BeforeLambdaBody: true
   BeforeWhile: true
-BreakBeforeBraces: Allman
+  IndentBraces: false
+BreakBeforeBraces: Custom

```

Comment:
Previous setting of `Allman` prevented `BraceWrapping` from taking effect. New `BraceWrapping` is identical to `Allman` settings but allows fixing of single line lambdas once a fix is applied to `BeforeLambdaBody` (see above).